docker/docker/docker-compose.yaml

```diff
@@ -0,0 +1,135 @@
+# To start docker-compose:
+# 1. Copy the example .env file:
+#    cp ../docker/webapi/.env.example ../docker/webapi/.env
+# 2. Edit the .env file to add the corresponding configuration values.
+# 3. Copy the example .env file for the web searcher plugin:
+#    cp ../docker/plugins/web-searcher/.env.example ../docker/plugins/web-searcher/.env
+# 4. Edit the .env file for the web searcher plugin to add the corresponding configuration values.
+# 5. With the .env files populated, you can now start docker-compose by running "docker-compose up --build"
+
+version: "3"
+services:
+  chat-copilot-webapp-nginx:
+    image: chat-copilot-webapp-nginx
+    build:
+      context: ..
+      dockerfile: docker/webapp/Dockerfile.nginx
+      args:
+        - REACT_APP_BACKEND_URI=http://localhost:3080
+    ports:
+      - 3000:3000
+    depends_on:
+      chat-copilot-webapi:
+        condition: service_started
+  #  chat-copilot-webapp:
```

Comment:
Why keep this commented stuff?

Reply:
the default setting is using webapi nginx dockerfile, while I leave commented stuff to give the flexibility to use webapi dockerfile.